checks/libs/win/winpdh.py

```diff
@@ -1,18 +1,44 @@
 import time
 import win32pdh
+import _winreg
 
 DATA_POINT_INTERVAL = 0.01
 
 class WinPDHCounter(object):
+    # store the dictionary of pdh counter names
+    pdh_counter_dict = {}
+
     def __init__(self, class_name, instance_name):
-        self._class_name = class_name
-        self._instance_name = instance_name
+        self._get_counter_dictionary()
+        self._class_name = win32pdh.LookupPerfNameByIndex(None, int(WinPDHCounter.pdh_counter_dict[class_name]))
+        self._instance_name = win32pdh.LookupPerfNameByIndex(None, int(WinPDHCounter.pdh_counter_dict[instance_name]))
         self.hq = win32pdh.OpenQuery()
 
     def __del__(self):
         if(self.hq):
             win32pdh.CloseQuery(self.hq)
 
+    def _get_counter_dictionary(self):
+        if WinPDHCounter.pdh_counter_dict:
+            # already populated
+            return
+
+        try:
+            val, t = _winreg.QueryValueEx(_winreg.HKEY_PERFORMANCE_DATA, "Counter 009")
```

Comment:
Can we document what this _"Counter 009"_ thing is - maybe a url (is it this https://msdn.microsoft.com/en-us/library/windows/desktop/aa373181(v=vs.85).aspx ?)

Reply:
Yeah... that's windows for you.  009 is the locale code, but there's no definition for it.  The docs (I added to the comment) actually say to add "Counter 009" as the parameter :(